src/Components/Portfolio.js

```diff
@@ -84,7 +73,7 @@ function Portfolio(props) {
     apiClient.getStock(ticker, props.setCurrentStock, "detailed")
   }
 
-    if (props.loggedIn) {
+    if (props.user.loggedIn && stockList != null) {
```

Comment:
Really like the idea of passing an entire user object!  Good job!

Reply:
Thanks!